src/vm/memory/memory.zig

```diff
@@ -207,6 +207,69 @@ pub const Memory = struct {
     }
 };
 
+// Utility function to help set up memory for tests
+//
+// # Arguments
+// - `memory` - memory to be set
+// - `vals` - complile time structure with heterogenous types
+fn memoryInner(memory: *Memory, comptime vals: anytype) !void {
+    inline for (vals) |row| {
+        const firstCol = row[0];
+        const relo = Relocatable.new(firstCol[0], firstCol[1]);
```

Comment:
Small detail but the proper name for this is `const address = Relocatable.new(firstCol[0], firstCol[1]);` because this corresponds to the address in memory no?